portal/static/js/main.js

```diff
@@ -1868,7 +1868,15 @@ OrgTool.prototype.populateOrgsList = function(items) {
         };
         if (item.extension) orgsList[item.id].extension = item.extension;
         if (hasValue(item.language)) orgsList[item.id].language = item.language;
-        if (item.identifier) orgsList[item.id].identifier = item.identifier;
+        if (item.identifier) {
+            orgsList[item.id].identifier = item.identifier;
+            (item.identifier).forEach(function(identifier) {
+                if (identifier.system === "http://us.truenth.org/identity-codes/shortname") {
```

Comment:
That string might be better as a global variable/constant (if you're using it more than once)